src/wal/record.rs

```diff
@@ -15,13 +16,11 @@ pub enum RecordType {
     Middle = 4,
     Last = 5,
 }
+}
 
 impl RecordType {
     pub fn from_u8(i: u8) -> Option<RecordType> {
```

Comment:
It turns out that if I just delete this method then the compiler just picks the correct from_i32.